### DISCUSSION TITLE
Learn Dashboard app course

### DISCUSSION THREAD

**Discussion by usman26392** [vercel/community#5773] [Bug Report]
### Page to Investigate

https://nextjs.org/learn/dashboard-app/static-and-dynamic-rendering

### Steps to Reproduce

Hello Next js team, i am doing a course from the Next js website.  when I reached chapter 8 in which there is a problem in noStore() function how can I fix it ?
![error](https://github.com/vercel/community/assets/56800807/dc76c3eb-4f49-467e-bd69-33bb9adb24df)

**Comment by delbaoliveira** [comment]
Hey @MuhammadUsman-Usman, could you share more information about the error you're seeing?

**Comment by usman26392** [comment]
@delbaoliveira thank you for your response!
I switched the next chapter with the comment noStore() function, and after that, I implemented streaming(chapter) in my project I 
removed that comment from the function I realized that was fixed. I remember that there was a squiggly red line asking for parameters or definitions. I don't know how it was fixed.
